apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -72,20 +76,78 @@ const Indictment: React.FC = () => {
 
   useDeb(workingCase, ['indictmentIntroduction', 'demands'])
 
+  const setDriversLicenseSuspensionRequest = useCallback(
+    (indictmentCounts?: TIndictmentCount[]) => {
+      let requestDriversLicenseSuspension = false
+
+      const reduce = () =>
+        indictmentCounts?.forEach((count) =>
+          count.offenses?.forEach((offence) => {
+            switch (offence) {
+              case IndictmentCountOffense.DrivingWithoutLicence:
+                requestDriversLicenseSuspension = false
+                return
+              case IndictmentCountOffense.DrunkDriving:
+                if (
+                  count.substances &&
+                  count.substances[IndictmentCountOffense.DrunkDriving] &&
+                  count.substances[IndictmentCountOffense.DrunkDriving][
+                    'ALCOHOL'
+                  ] >= '1,20'
```

Comment:
Consider extracting this to a constant with a declarative name. I'm guessing this is the max blood alcohol limit in Iceland before you loose your licence?

Reply:
This is a constant in the SubstanceMap type but for some reason this code snippet did not pick that up. Will investigate further.